### ISSUE TITLE
"Search" menu entry

### ISSUE THREAD

**Issue by chaoran-chen** [MusicConnectionMachine/VisualizationG6#124]
Do we really need a link to "search" in the menu? We already have a search field in the header bar and on the landing page..

**Comment by pfent** [comment]
The home page is also kindof a search page, since there is a big central search field.

I'd vote for dropping that entry

**Comment by joern-vh** [comment]
I would support dropping the entry.

**Comment by kordianbruck** [comment]
I really don't mind it actually. But if you guys insist, then I'm not opposed.

**Comment by sacdallago** [comment]
I agree with @kordianbruck , I don't mind it. On the contrary the home page is a bit crowded, thus having an extra search section feels like a little safe space to me. ☁️ opinions

**Comment by syncall** [comment]
Keep in mind that on mobile devices the search bar is not there and the
user has to navigate to search somehow

Christian Dallago <notifications@github.com> schrieb am Di., 25. Apr. 2017,
23:57:
